test/runtests.jl

```diff
@@ -950,6 +950,7 @@ end
     @test ceil(u"m", (125//1)u"cm", sigdigits=2) === 1.3u"m"
     @test trunc(u"m", -125u"cm", sigdigits=2) === -1.2u"m"
     @test trunc(u"m", (125//1)u"cm", sigdigits=2) === 1.2u"m"
+    @test round(3.7001u"m", sigdigits=3) === 3.7u"m"
```

Comment:
```suggestion
    @test round(3.7001u"m", sigdigits=3) === 3.7u"m"
    @test round(3.7001u"m", RoundUp, sigdigits=3) === 3.71u"m"
```
Since specifying a `RoundingMode` is now allowed, it should be tested as well.